app/views/escorts/print/cover.html.slim

```diff
@@ -28,83 +28,83 @@ html
       div.grid.detainee
         div.grid-row.header-row
           hi
-            span#detainee-identifier.strong-text => detainee.identifier
-            = detainee.forenames
+            span#detainee-identifier.strong-text => identifier(escort.detainee)
+            = escort.detainee.forenames
         div.grid-row.indicators
           #not_for_release_alert.grid-column.column-one-half
-            = render partial: 'escorts/print/alert', locals: { alert_name: :not_for_release, on: alerts[:not_for_release] }
-            - if alerts[:not_for_release]
-              p.alert-text= move.not_for_release_text
+            = render partial: 'escorts/print/alert', locals: { alert_name: :not_for_release, on: escort.alerts[:not_for_release] }
```

Comment:
All these `render partial: 'foo', locals: {bar: 'bish', baz: 'bosh'}` calls could be shortened to `render 'foo', bar: 'bish', baz: 'bosh'`